pyproject.toml

```diff
@@ -1,25 +1,40 @@
-[tool.poetry]
```

Comment:
oh I probably have to do this in another PR first

Reply:
Btw, do you want me to do the PR for that ? Or you want to do it ?

Reply:
If you have the room and motivation to do it, feel free and I am happy to review it! At this point I am all over the place with a lot of integrations, so it would otherwise end up on my list somewhere

Reply:
Here we go https://github.com/joostlek/python-waqi/pull/492